gtest_parallel_tests.py

```diff
@@ -173,8 +174,19 @@ def test_run_task_basic(self):
         sorted(task.task_id for task in task_manager.failed),
         sorted(task.task_id for task in task_mock_factory.failed))
 
-    self.assertEqual(task_manager.global_exit_code, 1)
+    self.assertEqual(task_manager.global_exit_code, expected_exit_code)
+
+  def test_run_task_basic(self):
```

Comment:
Can you rename all of these tests to what they actually verify? Such as "test_running_without_failures_should_do_X" or something?